src/home/HomeCookieSnackbar.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {Button, Snackbar, IconButton, useMediaQuery} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import Slide, { SlideProps } from '@mui/material/Slide';
+import theme from '../theme';
+
+
+type CookieSnackbarProps = 
+{
+    timeToWait: number;
+}
+
+type TransitionProps = Omit<SlideProps, 'direction'>;
+
+function TransitionUp(props: TransitionProps) {
+    return <Slide {...props} direction="up" />;
+}
+
+const HomeCookieSnackbar = ({timeToWait} : CookieSnackbarProps) =>
+{
+    const isDesktop = useMediaQuery(theme.breakpoints.down('lg'));
```

Comment:
let's also account for md breakpoints (tablet sizes). the spacing does look a little funky when testing in tablet screens, the snackbar is hugging the right side